### PR TITLE
Add/readme storybooknative

### PR DESCRIPTION
**Pull request by RobinCsl**
<br/><br/><br/><url>LiveURL: https://orbit-components-add-README-storybooknative.surge.sh</url>

### CODE REVIEW

File: src/Portal/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default class Portal extends React.Component<Props> {
   constructor(props: Props) {
     super(props);
     if (typeof window !== "undefined") {
-      this.node = document.getElementById(this.props.element || "modal") || document.body;
+      this.node = this.props.element ? document.getElementById(this.props.element) : document.body;
```
Comment (tbergq) [inline src/Portal/index.js]: Why this change?

Comment (RobinCsl) [reply src/Portal/index.js]: This is from syncing with `master`

Comment (tbergq) [reply src/Portal/index.js]: Aha, ok, just wanted to make sure there were no accidental changes.

**Comment by RobinCsl** [comment]
I will close this one, and we will sync with master in another isolated PR.
